apis/python/node/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "dora-node-api-python"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+dora-node-api = { path = "../../rust/node" }
+pyo3 = "0.16"
+eyre = "0.6"
+pollster = "0.2"
```

Comment:
This dependency seems to be unused. And it looks very similar to the `block_on` function provided by the `futures` crate, so I don't think that we need this dependency.